Resources/config/messenger.xml

```diff
@@ -4,17 +4,12 @@
            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
            xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
     <services>
-        <service id="doctrine.orm.messenger.middleware_factory.transaction" class="Symfony\Bridge\Doctrine\Messenger\DoctrineTransactionMiddlewareFactory" public="false">
```

Comment:
This change breaks a test:
https://github.com/doctrine/DoctrineBundle/blob/d57c1a35cd32e6b942fdda90ae3888cc1bb41e6b/Tests/DependencyInjection/DoctrineExtensionTest.php#L700-L701
https://travis-ci.org/doctrine/DoctrineBundle/jobs/448961134

Reply:
tests fixed thanks for the direct links.